SDL/Event.cu

```diff
@@ -326,25 +329,28 @@ void SDL::Event::resetEvent()
             n_quintuplets_by_layer_endcap_[i] = 0;
         }
     }
-    if(hitsInGPU){cudaFreeHost(hitsInGPU);
+    if(hitsInGPU){//cms::cuda::free_host(hitsInGPU);
+    cudaFreeHost(hitsInGPU);
     hitsInGPU = nullptr;}
-    if(mdsInGPU){cudaFreeHost(mdsInGPU);
+    if(mdsInGPU){cms::cuda::free_host(mdsInGPU);
     mdsInGPU = nullptr;}
-    if(rangesInGPU){cudaFreeHost(rangesInGPU);
+    if(rangesInGPU){cms::cuda::free_host(rangesInGPU);
     rangesInGPU = nullptr;}
-    if(segmentsInGPU){cudaFreeHost(segmentsInGPU);
+    if(segmentsInGPU){cms::cuda::free_host(segmentsInGPU);
     segmentsInGPU = nullptr;}
-    if(tripletsInGPU){cudaFreeHost(tripletsInGPU);
+    if(tripletsInGPU){cms::cuda::free_host(tripletsInGPU);
     tripletsInGPU = nullptr;}
-      if(quintupletsInGPU){cudaFreeHost(quintupletsInGPU);
+      if(quintupletsInGPU){cms::cuda::free_host(quintupletsInGPU);
       quintupletsInGPU = nullptr;}
-    if(trackCandidatesInGPU){cudaFreeHost(trackCandidatesInGPU);
+    if(trackCandidatesInGPU){cms::cuda::free_host(trackCandidatesInGPU);
     trackCandidatesInGPU = nullptr;}
-    if(pixelTripletsInGPU){cudaFreeHost(pixelTripletsInGPU);
+    if(pixelTripletsInGPU){//cms::cuda::free_host(pixelTripletsInGPU);
+    cudaFreeHost(pixelTripletsInGPU);
```

Comment:
Is there a reason this isn't using cms::cuda::free_host?

Reply:
There are 5 variables or so that currently cause errors when using the host allocator. I know why "hitsInGPU" causes an error, but I have not looked too deeply into this variable yet. Since it will take a good chunk of time to go through each of these edge cases and find the best fix for each of them (and they probably won't substantially increase speedup), I think it makes sense to fix them in a later PR.

Reply:
ok makes sense

Reply:
Maybe if you provide a list of the variables that are being difficult @tresreid or @bsathian can give a hint why they are different than the others